web/src/styles/sections/nav-bar.scss

```
@@ -28,12 +28,31 @@ $icon-offset: 45px;
   z-index: 5;
 }
 
+.Mui-focusVisible {
+  &.nav-menu-item-container {
+    background-color: $transparent;
+  }
+
+  &.nav-menu-item-container.bg-primary-extra-light-mui-selected-override {
+    background-color: $primary-extra-light;
+  }
+
+  &.profile-menu-item {
+    background-color: #fbfdf9;
+  }
+}
```

This is a total nit, but if you wanted you could take this a little further.
```suggestion
.Mui-focusVisible {
  &.nav-menu-item-container {
    background-color: $transparent;
    
    &.bg-primary-extra-light-mui-selected-override {
        background-color: $primary-extra-light;
    }
  }

  &.profile-menu-item {
    background-color: #fbfdf9;
  }
}
```